test/cloudnative/cloudnative_test.go

```diff
@@ -34,4 +37,5 @@ func TestCloudNative(t *testing.T) {
 	testEnvironment.Test(t, Install(t, false))
 	testEnvironment.Test(t, CodeModules(t, false))
 	testEnvironment.Test(t, NetworkProblems(t))
+	testEnvironment.Test(t, WithProxy(t, proxy.ProxySpec))
```

Comment:
this should be a separate test-case like the `cloudnativeisito` as it requires special environment like `cilium` and a proxy
it could be `cloudnativeproxy` or something